recipes/tetools_l-modolo/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+sed -i 's|#!/usr/bin/Rscript|#!/usr/bin/env Rscript|' TEdiff.R
+sed -i 's|#!/usr/bin/python3|#!/usr/bin/env python|' TEcount.py
+sed -i 's|#!/usr/bin/python3|#!/usr/bin/env python|' PingPong.py
+EXEC='PingPong.py TEcount.py TEdiff.R UrQt pingpongpro/pingpongpro'
```

Comment:
UrQt is available as Bioconda package IIRC. Please pull it in from there (i.e. depend on package)

Reply:
What is pingpongpro? Should that be packaged separately?